lib/enigma.rb

```diff
@@ -68,6 +84,16 @@ def character_converter(message_increment, shift_parameters)
     end
   end
 
-
+  def character_converter_decrypt(message_increment, shift_parameters)
+    message_increment.map.with_index do |character, index|
+      if @character_list.include?(character)
+          rotated_list = @character_list.rotate(-(shift_parameters[index]))
```

Comment:
The ONLY difference between this and the encryption version is the minus in (-(shift_parameters[index])) for list rotation. this makes is shift left, rather than right.